ciaox11/deployment/handlers/ciaox11_deployment_handlers.mpc

```diff
@@ -1,16 +1,35 @@
 
+project (ciaox11_deployment_state) : install, \
+         ciaox11_lib, ciaox11_output, \
+         ciaox11_core, ciaox11_ccm_stub, \
+         ciaox11_logger {
+  dynamicflags += CIAO_DEPLOYMENT_STATE_BUILD_DLL
+  Source_Files {
+    ciaox11_state.cpp
+  }
+
+  Header_Files {
+    ciaox11_state.h
+  }
+
+  specific {
+    install_dir = ciaox11/deployment/handlers
+  }
+}
+
 project (ciaox11_deployment_handlers) : install, \
          dancex11_deployment_stub, ciaox11_lib, ciaox11_config, \
          ciaox11_output, ciaox11_core, ciaox11_ccm_session_stub, \
          dancex11_stub, dancex11_state, \
          ciaox11_logger, taox11_portableserver, \
          ciaox11_ccmobject_stub {
+  after += ciaox11_deployment_state
```

Comment:
Shouldn't this come through a base project?

Reply:
Not in the core itself.